client/lib/abtest/active-tests.js

```diff
@@ -107,4 +107,14 @@ module.exports = {
 		},
 		defaultVariation: 'original'
 	},
+	promoteFreeDomain: {
+		datestamp: '20160301',
+		variations: {
+			original: 50,
+			freeDomain: 50
+		},
+		defaultVariation: 'original',
+		excludeSitesWithPaidPlan: true
```

Comment:
I mentioned it because I don't think we should show this promotion if the user is already on Premium, as it will interfere with the "upgrade to Business" discount (and it's not really relevant anymore at that point, since you already have a plan).